src/pages/Scan.vue

```diff
@@ -144,6 +152,15 @@ export default {
             }, 2000);
         },
 
+        showQueued: function() {
+            this.spinner = false;
+            this.queued = true;
+            var self = this;
+            setTimeout(function(){
```

Comment:
I've not read this part of the code yet, I'll check to see if it's actually needed.